Doc/library/shutil.rst

```diff
@@ -706,17 +706,18 @@ provided.  They rely on the :mod:`zipfile` and :mod:`tarfile` modules.
 
    The keyword-only *filter* argument is passed to the underlying unpacking
    function. For zip files, *filter* is not accepted.
-   For tar files, it is recommended to set it to ``'data'``,
+   For tar files, it is recommended to use the default, ``'data'``,
    unless using features specific to tar and UNIX-like filesystems.
    (See :ref:`tarfile-extraction-filter` for details.)
-   The ``'data'`` filter will become the default for tar files
-   in Python 3.14.
 
    .. audit-event:: shutil.unpack_archive filename,extract_dir,format shutil.unpack_archive
 
    .. warning::
 
-      Never extract archives from untrusted sources without prior inspection.
+      Never extract archives from untrusted sources without prior inspection,
+      even when using the ``'data'`` filter, but especially if using the
+      ``'tar'`` or ``'fully_trusted'`` filters.
```

Comment:
There are only three filters for now and I think that an unstrusted source englobes the entire source, whether some parts are trusted or not. 

I mean, if I use 'fully_trusted', it contradicts the fact that it's assumed to be untrusted =/ (In summary, the current wording is fine to me).

Reply:
Good point, I made an update here:
57c60e7287823fd34d8e1365e40c63d34ca2fb40

I think this gets the point across without being awkward.

Reply:
Yes, that's better!